### PR TITLE
Add module definition to generated test files

### LINKED ISSUE #22: Add the definition of the module when creating a new test file
When we create a new test file it'll be nice to add a simple template in this file, like:
```ex
defmodule Foo.BarTest do
end
```

### PR DESCRIPTION
**Pull request by w-sanches**
Closes #22

After creating the new test file, the user will be presented an **unsaved** test file with the template provided in the issue. I'm getting the module name from the first `defmodule` found in the current file, which might not be ideal.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -13,15 +13,28 @@ function showConfirmationDialog(text, button) {
...
 
-function createNewTestFile(dir, file) {
+async function getModuleName(uriFile) {
+  const content = (await vscode.workspace.fs.readFile(uriFile)).toString();
+  const moduleDefinition = content.match(/defmodule (.*) do/);
+
+  return moduleDefinition[1];
+}
+
+async function createNewTestFile(dir, file) {
```
Comment (mracos) [inline extension.js]: Why this function is now `async`? 🤔

Comment (w-sanches) [reply extension.js]: Just so I can use `await` inside of it and avoid having to deal with nested `.then()`

Comment (mracos) [reply extension.js]: Okie dokie then 😄

Comment (w-sanches) [reply extension.js]: Did you mean `await Okie dokie` 😄 ?

**Pull request merged**